recipe/scripts/activate.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [[ `uname` == "Darwin" ]]; then
+    echo "Activating cling"
+    PRODUCT_VERSION=$(sw_vers -productVersion)
+    OSX_VERSIONS=( ${PRODUCT_VERSION//./ } )
+    OSX_VERSION_MAJOR="${OSX_VERSIONS[0]}"
+    OSX_VERSION_MINOR="${OSX_VERSIONS[1]}"
+    if [ "$OSX_VERSION_MAJOR" -ge "10" ]&& [ "$OSX_VERSION_MINOR" -ge "14" ]; then
+        export CLING_INCLUDE_PATHS=$(xcode-select -p)/SDKs/MacOSX.sdk/usr/include
```

Comment:
I don't think you need to detect versions here. This path didn't *move*, they just stopped copying the files into /usr/include in favor of having only the "original" xcode location, which has been around for some time.

Reply:
On 10.13, this include path does not exist, (while we do have bin, share etc in the parent directory)

Reply:
I am not sure that it is an issue for cling to have non-existing include directories.

Reply:
What I mean is that I don't think there's a reason to check for the mac version at all. Instead of "if mac 10.14 check here" user "if sdk/usr/include exists use it, else use /usr/include". Regardless of host OS version.